patientsearch/models/sync.py

```diff
@@ -1,10 +1,31 @@
 """Manages synchronization of Model data, between external and internal stores"""
+from copy import deepcopy
+
 from flask import abort, current_app
 from jmespath import search as json_search
 import requests
 from .bearer_auth import BearerAuth
 
 
+def add_identifier_to_resource_type(bundle, resource_type, identifier):
```

Comment:
Why is this only acting on the given `resource_type`? Shouldn't any given Bundle have all the same resourceTypes in it?

Reply:
Only as it stands right now.  A bundle can hold all kinds of resources - say observations, appointments, etc.  I chose to implement it this way to keep from polluting other objects should that kind of search bundle sneak in...